src/test/java/seedu/multitasky/storage/JsonUserPrefsStorageTest.java

```diff
@@ -16,36 +17,47 @@
 import seedu.multitasky.commons.util.FileUtil;
 import seedu.multitasky.model.UserPrefs;
 
+// @@author A0132788U
+
 public class JsonUserPrefsStorageTest {
 
-    private static final String TEST_DATA_FOLDER = FileUtil
-            .getPath("./src/test/data/JsonUserPrefsStorageTest/");
+    private static final String TEST_DATA_FOLDER = FileUtil.getPath("./src/test/data/JsonUserPrefsStorageTest/");
 
     @Rule
     public ExpectedException thrown = ExpectedException.none();
 
     @Rule
     public TemporaryFolder testFolder = new TemporaryFolder();
 
+    /**
+     * Reads a null file path to throw an exception
+     */
     @Test
-    public void readUserPrefs_nullFilePath_throwsNullPointerException() throws DataConversionException {
+    public void getNullPointer() throws DataConversionException {
```

Comment:
I think the original test names were actually more descriptive as to what the test is actually checking